roles/installer/templates/deployments/deployment.yaml.j2

```diff
@@ -20,8 +20,25 @@ spec:
       labels:
         {{ lookup("template", "../common/templates/labels/common.yaml.j2")  | indent(width=8) | trim }}
         {{ lookup("template", "../common/templates/labels/version.yaml.j2") | indent(width=8) | trim }}
-{% if annotations %}
       annotations:
+{% for template in [
+    "configmaps/config",
+    "secrets/app_credentials",
+    "storage/persistent",
+  ] %}
+        checksum-{{ template | replace('/', '-') }}: "{{ lookup('template', template + '.yaml.j2') | md5 }}"
```

Comment:
:heart: This is a great trick!  This is an elegant solution to the issue of deployments not being cycled when changes are made to the ConfigMap.